CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Release 0.19.1
```

Comment:
0.19.0 hasn't gone out yet.

Reply:
I'm staging this to ship after 0.19.0 ships.